examples/units/artist_tests.py

```diff
@@ -3,11 +3,11 @@
 Artist tests
 ============
 
-Test unit support with each of the matplotlib primitive artist types
+Test unit support with each of the Matplotlib primitive artist types.
 
-The axes handles unit conversions and the artists keep a pointer to
-their axes parent, so you must init the artists with the axes instance
-if you want to initialize them with unit data, or else they will not
+The axis handles unit conversions and the artists keep a pointer to
+their axis parent, so you must initialize the artists with the axis instance
```

Comment:
I'm thinking this is still a bit long; can you break the sentence in two: "their axis parent. You must initialize ..."?